app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java

```diff
@@ -120,13 +120,8 @@ public void init(int position, Contribution contribution) {
      */
     @Nullable
     private String chooseImageSource(String thumbUrl, Uri localUri) {
-        String imageSource = thumbUrl;
-        if (TextUtils.isEmpty(imageSource)) {
-            imageSource =
-                localUri != null ? localUri.toString()
-                    : null;
-        }
-        return imageSource;
+        return !TextUtils.isEmpty(thumbUrl) ? thumbUrl : (localUri != null ? localUri.toString()
```

Comment:
I am a huge nitpicker, I apologise.

For these scenarios I generally put each value that is returned on its own line

```
condition ? value :
  condition ? value :
    value
```

feel free to ignore

Reply:
Fair point, added :)